lib/twiml/VoiceResponse.d.ts

```diff
@@ -205,6 +205,14 @@ declare namespace VoiceResponse {
 
   type ConferenceTrim = 'trim-silence'|'do-not-trim';
 
+  type ConversationEvent = 'call-initiated'|'call-ringing'|'call-answered'|'call-completed';
```

Comment:
Where are these additional types coming from?

Reply:
It was added to api-def 4 months ago. I generated the changes based on the updated api-def, hence the two different commits. 

https://code.hq.twilio.com/twilio/api-definitions/blame/master/twiml/voice/conversation.json#L24-L29